connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -133,6 +134,10 @@ object CheckConnectJvmClientCompatibility {
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.DataFrameReader.jdbc"),
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.DataFrameWriter.jdbc"),
 
+      // DataFrameStatFunctions
+      ProblemFilters.exclude[Problem]("org.apache.spark.sql.DataFrameStatFunctions.bloomFilter"),
+      ProblemFilters.exclude[Problem]("org.apache.spark.sql.DataFrameStatFunctions.this"),
```

Comment:
Please also remove `ProblemFilters.exclude[Problem]("org.apache.spark.sql.Dataset.na")`

Reply:
Are you want to  say `org.apache.spark.sql.Dataset.stat`?  `na` is `DataFrameNaFunctions`, not `DataFrameStatFunctions`

Reply:
yes you are right.... :)

Reply:
done